projects/aca-shared/src/lib/components/open-in-app/open-in-app.component.scss

```diff
@@ -15,3 +15,7 @@
 .open-in-app.mat-button {
   overflow-x: hidden;
 }
+
+.open-in-app.cdk-program-focused .mat-button-focus-overlay {
+  opacity: 0 !important;
```

Comment:
I'm not sure, but I think it's recommended not to use !important

Reply:
hi Mykyta, I checked without !important, but it's not overridding the property.

Reply:
@jatin2008 could you please send a screenshot with browser's console including HTML and CSS to show how it looks without !important? Maybe there is some way but please include that so I may suggest something.

Reply:
yeah sure alek, i will share screenshot here

with !important

![Screenshot from 2023-02-27 16-11-54](https://user-images.githubusercontent.com/105338943/221542687-c4de5a9a-ba7e-4c9b-9560-36fdfe9a3596.png)

Without !important

![Screenshot from 2023-02-27 16-11-34](https://user-images.githubusercontent.com/105338943/221542776-6f049873-6381-4300-9b48-162e972c1b8d.png)

Reply:
Thank you @jatin2008. Looking at styles I guess that your selector priority is just too weak. Can you try for example following selector: 
```scss
.open-in-app.mat-button.cdk-program-focused .mat-button-focus-overlay {
  opacity: 0;
}
```
Thanks that your selector will have one more class so should have a bit higher priority (it has now extra .mat-button)

Reply:
Without above change it has the same number classes as selector for .12 so it is why it could take as priority .12.

Reply:
yess now it is working without !important by using the higher priority code above.

Reply:
thanks